Hairtrigger-Bunnies/nodejs-bangazon-site
# Bobby features

Changed:
-Added pug for 'add product' to sell form
-Added route to get to new product form
-Added function in controller to render new product form

Because:
-You can't view the new product form without the view, route, and controller I expanded on

To test:
(steps to test -- include everything!)
1. Run ```npm start``` 
2. Go to ```localhost:8080``` and sign in or register user
3. Once authenticated, enter in ```http://localhost:8080/product/add``` in the browser to see new form for creating a product to sell
